components/DOBInput/story.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import moment from "moment"
+import DOBInput from "./"
+
+import { boolean, text } from "@storybook/addon-knobs"
+
+import 'react-datetime/css/react-datetime.css'
+
+export default function Story(stories) {
+  stories.add("DOBInput", () => {
+    const props = {
+      preventMinor: boolean("Prevent Minor", true),
+      viewDate: text("Default Calendar Date", moment("2000-01-01"))
```

Comment:
A hard-coded "2001-01-01" won't be 18 years ago next year. I know this is just for demonstration, but a more accurate method would be `today - 18.years`.

Reply:
ah, that was not supposed to be there. I was making sure one could still override the viewDate option. DOBInput is a decorator component and should not hide the underlying DatePicker's API.